src/hermit.rs

```diff
@@ -0,0 +1,21 @@
+use crate::Error;
+use core::{cmp::min, mem::MaybeUninit, num::NonZeroU32};
+
+extern "C" {
+    fn sys_read_entropy(buffer: *mut u8, length: usize, flags: u32) -> isize;
+}
```

Comment:
The newest version of [`hermit-abi`](https://crates.io/crates/hermit-abi) (docs are currently broken) already includes this definition.

Reply:
In such simple cases we prefer to link directly instead of adding a dependency on a "system" crate.